components/footer/Footer.style.js

```diff
@@ -7,7 +7,6 @@ const FooterContainer = styled.div`
   width: 100%;
```

Comment:
Unrelated nit: Footer links are not working

Reply:
fixed